### PR TITLE
chore(react-chart): split react and chart demos

### PR DESCRIPTION
**Pull request by Krijovnick**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,9 +32,9 @@
     "build:ts:watch": "nodemon --exec \"yarn build:ts\" --watch \"packages/*/docs/reference/*.md\" --watch \"scripts/ts-generator.js\" --ext md,js",
     "build:watch": "concurrently \"yarn build:ts:watch\" \"lerna run --parallel build:watch\"",
     "build:site": "yarn run build:site:demos && yarn run build:site:docs",
-    "build:site:demos": "yarn run build:ts && lerna run build --ignore @devexpress/dx-react-demos && lerna run build:prod --scope @devexpress/dx-react-demos",
+    "build:site:demos": "yarn run build:ts && lerna run build --ignore @devexpress/dx-react-grid-demos && lerna run build:prod --scope @devexpress/dx-react-grid-demos",
```
Comment (alexey-semikozov) [inline package.json]: Why ignore only grid demos?

File: packages/dx-react-chart-demos/package.json [hunk 2]
```diff
@@ -0,0 +1,85 @@
+{
+  "private": true,
+  "name": "@devexpress/dx-react-chart-demos",
+  "version": "1.3.0",
```
Comment (alexey-semikozov) [inline packages/dx-react-chart-demos/package.json]: update version, plz

File: packages/dx-react-grid-demos/src/demo-sources/demo.test.jsxt [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+jest.spyOn(Element.prototype, 'getBoundingClientRect').mockImplementation(() => ({
+  top: 0,
+  left: 0,
+  width: 800,
+  height: 600,
+  right: 800,
+  bottom: 600,
+}));
+
+Element.prototype.getBBox = () => {};
```
Comment (alexey-semikozov) [inline packages/dx-react-grid-demos/src/demo-sources/demo.test.jsxt]: For grid, it is redundancy method

File: gulpfile.js [hunk 4]
```diff
@@ -132,22 +132,28 @@ gulp.task('site:docs', function() {
...
-  return gulp.src(['packages/dx-react-demos/dist/*'])
-    .pipe(gulp.dest(distPath + 'react/demos/dist/'));
+gulp.task('site:demos:react:grid', function() {
+  return gulp.src(['packages/dx-react-grid-demos/dist/*'])
+    .pipe(gulp.dest(distPath + 'react/grid-demos/dist/'));
+});
+
+gulp.task('site:demos:react:chart', function() {
+  return gulp.src(['packages/dx-react-chart-demos/dist/*'])
+    .pipe(gulp.dest(distPath + 'react/chart-demos/dist/'));
```
Comment (kvet) [inline gulpfile.js]: Use 'chart/demos' instead

File: gulpfile.js [hunk 5]
```diff
@@ -132,22 +132,28 @@ gulp.task('site:docs', function() {
...
 });
 
 gulp.task('site:demos:vue', function() {
   return gulp.src(['packages/dx-vue-demos/dist/*'])
-    .pipe(gulp.dest(distPath + 'vue/demos/dist/'));
+    .pipe(gulp.dest(distPath + 'vue/grid-demos/dist/'));
```
Comment (kvet) [inline gulpfile.js]: Use 'grid/demos' instead

File: package.json (hunk 1 above)
Comment (Krijovnick) [reply package.json]: I'll fix it.

**Comment by Krijovnick** [comment]
@kvet already done

**Comment by Krijovnick** [comment]
@gsobolev I've removed this folder. But it can be recreated after the move into other branches where it is and stay here until you delete it yourself.

**Comment by gsobolev** [comment]
@Krijovnick, I see your point, but I found this folder in your branch [here](https://github.com/Krijovnick/devextreme-reactive/tree/demos/packages/dx-react-demos/src/demo-sources/chart-basic)

**Comment by Krijovnick** [comment]
@gsobolev Oh, I see. I'll fix it.

**Pull request merged**